Update Beta Docs “introduction” (#152)
sourcecred/docs: docs/beta/introduction.md

So how does an algorithm determine how much someone's work is worth in the bigger picture of a project? Sounds a little dystopian right? The algorithm uses what we call "Weights" which are a set of rules stating which kinds of contributions are worth what amounts of Cred. The Weights fo Cred distribution are decided by each community individually, thereby giving a community the opportunity to decide what it values and which actions uphold those values as they work together to create something. Every community using SourceCred can decide for themselves what is valuable and deserving of Cred, and what is not. 

Cred is "non-transferable" in that you cannot sell your Cred, or buy someone else's Cred. It is solely a representation of where value in the project has been created.

> If you want to dig deeper into the nuances of Cred, check out our [How Cred Works](https://sourcecred.io/docs/beta/cred) documentation. 
> 

Reviewer: Suggest using a relative link instead.
```suggestion
If you want to dig deeper into the nuances of Cred, check out our [How Cred Works](./cred) documentation. 

```

##### Ways you might end up with Grain in a project that's using SourceCred:

* You worked directly as a contributor, earning Cred -and therefore Grain- in that project.
* You bought Grain with another valuable currency (like ETH or Dollars) from a contributor in that project. 

So how is Grain useful? The transferability of Grain makes it a powerful tool in a number of different ways depending on how your project and/or community is operating. Perhaps you have funding already, but don't know how to distribute it. Maybe you'd like to use it as a tool for governance. Or maybe you'd like a way to allow people outside the project to financially support what you're doing by buying the Grain.

Part of what makes Grain (and SourceCred) so powerful is that it can be applied in many different ways based on the needs and parameters of the community. 

##### Examples of how a project might use Grain:

* Paying contributors with grants or funds it already has based on Cred/Grain.

> * Creating a project-specific currency that can be traded for other forms of currency (ie: ETH or US Dollars). 

Reviewer: ```suggestion
* Creating a project-specific currency that can be traded for other forms of currency e.g.: ETH or US Dollars). 
```

* Creating an internal currency that could be used solely for governance or to show support for features/ideas. 

##### Examples of how an individual might use Grain:

* Selling their grain back to the project in exchange for money from the project's budget (essentially wages you can claim by selling your Grain). 

> * "Boosting" specific initiatives, ideas, goals, etc in the project by "burning" some of their Grain to make more Cred for that initiative. And creating a small flow of Cred from the initiative back to themselves as a speculation bonus.

Reviewer: ```suggestion
* "Boosting" specific initiatives, ideas, goals, etc in the project by "burning" some of their Grain to make more Cred for that initiative. This creates a small flow of Cred from the initiative back to themselves, as a reward for sponsoring it.
```
